nixos/modules/services/misc/gitea.nix

```diff
@@ -162,6 +162,26 @@ in
             <manvolnum>7</manvolnum></citerefentry>.
           '';
         };
+
+        backupDir = mkOption {
+          type = types.str;
+          default = "${cfg.stateDir}/dump";
+          description = "Path to the dump files.";
+        };
+      };
+
+      ssh = {
+        enable = mkOption {
+          type = types.bool;
+          default = true;
+          description = "Enable SSH feature.";
```

Comment:
What exactly does this do? Does this turn on the built-in ssh server `gitea` has?

Reply:
No use external SSH server, option need to correct seeting and works with NAT - example - `sshPort` changed dislayed port to 2222, and on the router configuring dnat 2222 -> 22.

Reply:
Update description.